### DISCUSSION TITLE
[BUG] Reports with modern search not working

### DISCUSSION THREAD

**Discussion by HeKr09** [microsoft-search/pnp-modern-search#2674] [Q&A] [answered]
**Version used**
4.6.1

**Describe the bug**
Actually I don't know if this is a real bug but I don't find my question answered here. Therefor I created an issue.
A customer is using the PnP Modern Search for several months (2 years I guess).
Now he wanted to check the usage reports from the Microsoft Search but the reports were nearly empty which can't be possible because there were several requests.

**To Reproduce**
1. Install PnP Modern Search
2. Search for topics
3. Check the reports in Microsoft Search

**Expected behavior**
I thought that the queries will also be logged within the usage reports?

**Desktop (please complete the following information):**
 - Browser: Edge

**Comment by kasperbolarsen** [comment]
As per https://learn.microsoft.com/en-us/microsoftsearch/usage-reports I doubt that PnP modern search usage is expected to show up in the stats. I can see why it could be useful to include those searches into the stats but where to draw the line? If I am executing a search using the Graph Explorer or the REST API should they be included?

**Comment by gretchunkim** [comment]
@HeKr09 Instruct the customer to toggle off (disable) the "New usage analytics" from the dashboard Search & intelligence dashboard and try again to pull the data.  Once the data is shown then toggle back on.  I don't think this is a bug on the PnP search webpart but an oversight on  Microsoft Search>New analytics feature side.  I tested on my side and my search queries are still showing up fine. 

![image](https://user-images.githubusercontent.com/8564035/209273618-a5f94bb7-aee5-4352-bc4b-d12f857580fb.png)

**Comment by Jeroen00** [comment]
Still not clear to me if searches done through PnP search box and search results are or are not included in the Microsoft Search analytics insights (https://tenant.sharepoint.com/_layouts/15/SiteAdmin.aspx#/). @wobba could you please confirm?

**Reply by JoeAyre** [reply]
Hi @Jeroen00 ,

I don't believe searches performed via the modern search box web part are included in the 'Microsoft 365 Search and intelligence' analytics. See this discussion: https://github.com/microsoft-search/pnp-modern-search/discussions/1462 - it's from a couple of years ago but as far as I can tell, nothing has changed. @wobba - is there any update on this?

Thanks,
Joe

**Reply by wobba** [reply, marked as answer]
@Jeroen00 API calls are not recorded in analytics today.
